packages/sgugpushdpopd/SOURCES/popd

```diff
@@ -0,0 +1,4 @@
+#!/usr/sgug/bin/bash
+DIR_STACK=${DIR_STACK#* }
```

Comment:
re @Jenna64bit's error -- maybe invoke `cd ..` -- it will be at least reliable. Can one of you try this? I just ran into the same issue on my end with another package and this seems like it's 3/4 across the line.